tutorials/external-metastore-for-dataproc/index.md

```diff
@@ -112,10 +113,10 @@ Create a MySQL on Cloud SQL instance to be used later to host the Hive metastore
 
 In Cloud Shell, execute the following command to create a new Cloud SQL instance:
         
-        gcloud sql instances create hive-metastore-db \
-        --database-version="MYSQL_5_7" \
-        --activation-policy=ALWAYS \
-        --zone ${ZONE}
+    gcloud sql instances create hive-metastore-db \
```

Comment:
To create an instance with HA in the two zones where metastore clusters have been provisioned
- https://cloud.google.com/sql/docs/mysql/configure-ha

```
gcloud sql instances create hive-metastore-db \
     --database-version="MYSQL_5_7" \
     --activation-policy=ALWAYS \
     --enable-bin-log \
     --availability-type=regional \
     --zone ${ZONE_METASTORE_1} \
     --secondary-zone ${ZONE_METASTORE_2}
```

Reply:
Best practice would be to create the Cloud SQL instance without a public ip address.  To do this, these steps would be required:

```
     gcloud services enable servicenetworking.googleapis.com

     export VPC_NETWORK_NAME=default

     gcloud compute addresses create google-managed-services-${VPC_NETWORK_NAME} \
     --global \
     --purpose=VPC_PEERING \
     --prefix-length=16 \
     --network=projects/${PROJECT}/global/networks/${VPC_NETWORK_NAME}


     gcloud services vpc-peerings connect \
     --service=servicenetworking.googleapis.com \
     --ranges=google-managed-services-${VPC_NETWORK_NAME} \
     --network=${VPC_NETWORK_NAME} \
     --project=${PROJECT}


     gcloud beta sql instances create hive-metastore-db-1 \
     --database-version="MYSQL_5_7" \
     --activation-policy=ALWAYS \
     --enable-bin-log \
     --no-assign-ip \
     --network=projects/${PROJECT}/global/networks/${VPC_NETWORK_NAME} \
     --availability-type=regional \
     --zone ${ZONE_METASTORE_1} \
     --secondary-zone ${ZONE_METASTORE_2}
 
```

Reply:
done. updated in two separate sections/